src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -104,6 +104,18 @@ ScenesServer & ScenesServer::Instance()
 }
 void ReportAttributeOnAllEndpoints(AttributeId attribute) {}
 
+class ScenesClusterFabricDelegate : public chip::FabricTable::Delegate
+{
+    void OnFabricRemoved(const FabricTable & fabricTable, FabricIndex fabricIndex) override
+    {
+        SceneTable * sceneTable = scenes::GetSceneTableImpl();
+        VerifyOrReturn(nullptr != sceneTable);
+        sceneTable->RemoveFabric(fabricIndex);
```

Comment:
Will that perform any notifications with the fabric index (e.g. updating attributes), or in any other way assume the fabric index refers to a fabric in the fabric table?

Reply:
No, this only deletes scenes associated to a fabric and the structure that maps them from the flash. Comment added to SceneTable.h and scenes-server.cpp